### PR TITLE
Adhering to Kotlin Style Guide

### PR DESCRIPTION
**Pull request by neil-power-hs**
Adhering to the Kotlin Style Guide
Updating libraries and kotlin version.

### CODE REVIEW

File: src/main/kotlin/com/hootsuite/hermes/models/HermesModels.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.hootsuite.hermes.models
+
+/**
+ * DTO Object for a Hermes User
+ */
+data class UserDTO(
```
Comment (benjamin-hergert-hs) [inline src/main/kotlin/com/hootsuite/hermes/models/HermesModels.kt]: Come to think of it, I'm not a big fan of the `DTO` postfix. Are you worried about name clashes?

Comment (neil-power-hs) [reply src/main/kotlin/com/hootsuite/hermes/models/HermesModels.kt]: yes but I have refactored

**Comment by neil-power-hs** [comment]
!hermes

File: src/main/kotlin/com/hootsuite/hermes/model/Dto.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.hootsuite.hermes.model
+
+/**
+ * DTO Object for a Hermes User
```
Comment (benjamin-hergert-hs) [inline src/main/kotlin/com/hootsuite/hermes/model/Dto.kt]: Data Transfer Object Object

**Pull request merged**